### ISSUE TITLE
Error when fully qualifying a view with catalog name in CREATE/DROP VIEW in SQL Server

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#10595]
There seems to be a limitation in SQL Server regarding the qualification of view names in `CREATE / ALTER VIEW` statements:

```
Caused by: java.sql.BatchUpdateException: 'CREATE/ALTER VIEW' does not allow specifying the database name as a prefix to the object name.
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeBatch(SQLServerStatement.java:1845)
	at org.jooq.tools.jdbc.DefaultStatement.executeBatch(DefaultStatement.java:112)
	at org.jooq.tools.jdbc.DefaultStatement.executeBatch(DefaultStatement.java:112)
	at org.jooq.impl.BatchMultiple.execute(BatchMultiple.java:114)
```

This should be easy to work around.

**Comment by lukaseder** [comment]
We've already done this for similar sequence related commands

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.14.0. Backport scheduled for 3.13.5 (#10596)
